### PR TITLE
NSB101 snippet fixes

### PR DESCRIPTION
**Pull request by DavidBoike**

### CODE REVIEW

File: tutorials/nservicebus-101/lesson-2/Solution/ClientUI/Program.cs [hunk 1]
```diff
@@ -39,6 +37,8 @@ await endpointInstance.Stop()
 
         static async Task RunLoop(IEndpointInstance endpointInstance)
         {
+            var logger = LogManager.GetLogger<Program>();
```
Comment (SimonCropp) [inline tutorials/nservicebus-101/lesson-2/Solution/ClientUI/Program.cs]: @DavidBoike this goes against our advice that loggers should be static. People should never do this in a production system

**Comment by DavidBoike** [comment]
Sorry, I had merged thinking these were very minor fixes. In this case they are essentially single instances within that class, but you're right, people could get that idea. I'll have to prepare a new PR that goes back to static loggers and covers creating it within the tutorial. (That's what was missing before.)

**Comment by SimonCropp** [comment]
@DavidBoike thanks. u can link here in the new PR https://docs.particular.net/nservicebus/logging/#when-to-configure-logging

**Comment by SimonCropp** [comment]
sorry wrong link https://docs.particular.net/nservicebus/logging/usage

**Pull request merged**